gen_dockerfiles

```diff
@@ -106,7 +106,8 @@ RUN set -x \\
 	&& export GNUPGHOME="\$(mktemp -d)" \\
 	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys DD95CC430502E37EF840ACEEA5D32F012649A5A9 \\
 	&& gpg --export DD95CC430502E37EF840ACEEA5D32F012649A5A9 > /etc/apt/trusted.gpg.d/neurodebian.gpg \\
-	&& rm -r "\$GNUPGHOME"
+	&& rm -rf "\$GNUPGHOME" \\
+	&& apt-key list | grep neurodebian
```

Comment:
Oh right, I also added this as a minor sanity check -- `apt-key list` _will_ fail if any of the keys in the trusted directories are invalid. :+1: